src/plugin/jquery-plugin.js

```diff
@@ -256,9 +256,9 @@
      *   Describes layers added to the map
      * @property {boolean} [autoresize=true]
      *   Resize the map on <code>window.resize</code> (initialization only)
-     * @property {string} [tileServer]
+     * @property {string} [tileUrl]
      *   The open street map tile server spec default:
-     *   <code>http://tile.openstreetmap.org/<zoom>/<x>/<y>.png</code>
+     *   <code>http://tile.openstreetmap.org/&lt;zoom>/&lt;x>/&lt;y>.png</code>
```

Comment:
<code>http://tile.openstreetmap.org/&lt;zoom>/&lt;x>/&lt;y>.png</code>

shouldn't it be:
 <code>http://tile.openstreetmap.org/&lt;zoom&gt;/&lt;x>/&lt;y&gt;.png</code>

but it may be that I missing something.